best-html-reporter.js

```diff
@@ -9,16 +9,25 @@
  *   screenshotCB: callback function for taking and storing the screenshots.
  */
 
-var _ = require('underscore');
+var _ = require('underscore'),
+   fs = require('fs-extra');
 
 exports.init = function(config) {
   
   var results = {},
     currentSuite;
   
   config.screenshots = config.screenshots || 'none';
-  config.reportDir = config.reportDir || __dirname + '/reports'
-  
+
+  if (!config.reportDir) {
+    fs.pathExists('../reports').then(function(exists) {
```

Comment:
why ../reports? better to use the absolute path __dirname + '/reports'

Instead of the check and create, fs-extra provides mkdirs() that will check and create the dir if not present. please use that.

Reply:
but in console it shows like the reports path already exists warning !!.. That's why i have added that raj..